packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -562,7 +563,11 @@ void main() {
             height: 100.0,
             child: AndroidView(
               viewType: 'webview',
-              gestureRecognizers: <OneSequenceGestureRecognizer> [VerticalDragGestureRecognizer()],
+              gestureRecognizers: <Factory<OneSequenceGestureRecognizer>> [
```

Comment:
here and elsewhere: extraneous space before `[`

Reply:
done